admin/nodes/urls.py

```diff
@@ -3,5 +3,10 @@
 from . import views
 
 urlpatterns = [
-    url(r'^$', views.NodeFormView.as_view(), name='node'),
+    url(r'^$', views.NodeFormView.as_view(),
+        name='search'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/$', views.NodeView.as_view(),
+        name='node'),
+    url(r'^id-(?P<node_id>[a-z0-9]+)/remove_user-(?P<user_id>[a-z0-9]+)/$',
```

Comment:
It has become pretty standard. These can be removed, but I would rather remove them all on one PR rather than one of the 13- or 14 open ones. @mfraezz @sloria